### PR TITLE
Disable debug print if not debugging

### PR DESCRIPTION
**Pull request by Python1320**

### CODE REVIEW

File: lua/carrier/packages/carrier/_ctor.lua [hunk 1]
```diff
@@ -36,6 +36,7 @@ end
 
 local debugColor = Color (192, 192, 192)
 function Carrier.Debug (message)
+	if Carrier.IsLocalDeveloperEnabled and not Carrier.IsLocalDeveloperEnabled () then return end
```
Comment (notcake) [inline lua/carrier/packages/carrier/_ctor.lua]: Carrier.IsLocalDeveloperEnabled is defined in developer.lua

**Review by notcake — CHANGES_REQUESTED** [review]

File: lua/carrier/packages/carrier.bootstrap/carrier.bootstrap.lua [hunk 2]
```diff
@@ -4211,6 +4211,7 @@ end
 
 local debugColor = Color (192, 192, 192)
 function Carrier.Debug (message)
+	if not Carrier.IsLocalDeveloperEnabled () then return end
```
Comment (notcake) [inline lua/carrier/packages/carrier.bootstrap/carrier.bootstrap.lua:4318]: This is computer generated code. You have to rerun Carrier.Bootstrap.Generator.App

**Review by notcake — CHANGES_REQUESTED** [review]

File: lua/carrier/packages/carrier/_ctor.lua (hunk 1 above)
Comment (Python1320) [reply lua/carrier/packages/carrier/_ctor.lua]: Are we guaranteed to have IsLocalDeveloperEnabled by loading order?

File: lua/carrier/packages/carrier.bootstrap/carrier.bootstrap.lua (hunk 2 above)
Comment (Python1320) [reply lua/carrier/packages/carrier.bootstrap/carrier.bootstrap.lua:4318]: I could not figure out how to do this. Please elaborate.

File: lua/carrier/packages/carrier/_ctor.lua (hunk 1 above)
Comment (notcake) [reply lua/carrier/packages/carrier/_ctor.lua]: Yes

File: lua/carrier/packages/carrier.bootstrap/carrier.bootstrap.lua (hunk 2 above)
Comment (notcake) [reply lua/carrier/packages/carrier.bootstrap/carrier.bootstrap.lua:4318]: `luajit ./lua/carrier/bootstrap.luajit.lua Carrier.Bootstrap.Generator.App`

Comment (Python1320) [reply lua/carrier/packages/carrier.bootstrap/carrier.bootstrap.lua:4318]: Correct method and _folder_ seems to be 
`carrier/lua$ luajit ./carrier/bootstrap.luajit.lua Carrier.Bootstrap.Generator.App`

**Pull request closed without merging**